app/geant-exporter/ImportProcessConverter.cc

```diff
@@ -436,11 +442,37 @@ void ImportProcessConverter::store_multiple_scattering_tables(
     {
         if (G4VEmModel* model = process.GetModelByIndex(i))
         {
+            if (i > 1)
+            {
+                // Index is beyond the code scope, which only includes a low
+                // and high msc energy model
+                CELER_LOG(error)
+                    << "Cannot store multiple scattering table for process "
+                    << process.GetProcessName() << ", model "
+                    << process.GetModelByIndex(i)->GetName()
+                    << ". Model index is > 1.";
```

Comment:
Should probably be a warning and then add a  `continue` statement so it doesn't add the table